components/gcp/ml_engine/ml_engine/delete_version_op.py

```diff
@@ -0,0 +1,60 @@
+import json
+import logging
+
+from googleapiclient import errors
+
+from kfp.gcp.core.base_op import BaseOp
+from kfp.gcp.core import name_utils
+from ml_engine import utils
+
+class DeleteVersionOp(BaseOp):
+
+    def __init__(self, project_id, model_name, version_name, staging_dir, wait_interval):
+        super().__init__()
```

Comment:
Not super(DeleteVersionOp, self).__init__()?

Reply:
There is no need for the longer syntax here (and almost everywhere).